Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
@thinkyhead Reading through this, it seems most of the issues being discussed are addressed in that patch I submitted.  In regards to action of turning the encoder in relationship to contrast, my thoughts are regardless of how it works with the menus, turning clockwise should increase contrast, counter clockwise decrease.  That is how most controls like this work.  Clockwise to turn up the volume. Increase the temperature on a stove. (at least in the US).